cli/cmd/up/up.go

```diff
@@ -185,3 +214,20 @@ func getCurrentDir() string {
 	dir := filepath.Base(workingDir)
 	return strings.ToLower(dir)
 }
+
+func setObjLabels(objs []runtime.Object, s *riov1.Stack) ([]runtime.Object, error) {
+
+	for _, obj := range objs {
+
+		obj := obj.(metav1.Object)
```

Comment:
To do a safe programming make sure to do
`obj, err := obj.(metav1.Object)` in case obj is not a metav1.Object and code won't panic.